packages/components/Button/package.json

```diff
@@ -31,7 +31,7 @@
     "@uifabricshared/foundation-settings": ">=0.8.0 <1.0.0"
   },
   "devDependencies": {
-    "@office-iss/react-native-win32": "0.62.3",
+    "@office-iss/react-native-win32": "0.62.6",
```

Comment:
Like the other comment, these probably should be deps (unless we only use the typings and want to keep them opaque)